packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -21,7 +22,7 @@
 from PIL import Image
 from PIL.PngImagePlugin import PngInfo
 
-from js import FontFace, ImageData, XMLHttpRequest, document, window
+from js import FontFace, ImageData, document
```

Comment:
Maybe would be good to produce a better error message if this fails:
```py
try:
	from js import FontFace, ImageData, document
except ImportError:
	raise ImportError("html5canvasbackend is only supported in the browser in the main thread")
```

Reply:
Thanks, applied!